src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -120,17 +160,28 @@ public VariantContext merge(final List<VariantContext> vcs, final Locatable loc,
                 .computeEndFromAlleles(nonSymbolicAlleles(allelesList), loc.getStart(), loc.getStart())
                 .genotypes(genotypes).unfiltered()
                 .attributes(new TreeMap<>(attributes)).log10PError(CommonInfo.NO_LOG10_PERROR);  // we will need to re-genotype later
-
+        if (doSomaticMerge) {
+            if (aggregatedFilters.isEmpty()) {
+                    builder.passFilters();
+            } else {
+                if (!sawPassSample) {
+                    builder.filters(aggregatedFilters);  //if all samples are filtered, this will apply all those filters to the VCF
+                } else {
+                    builder.passFilters();
```

Comment:
Does this say that if any sample is unfiltered, we pass the variant?  That seems kind of liberal for artifacts, although I agree that it seems like the right thing to do for TLOD filtering.  Mutect2 has some new code for multi-sample filtering that tries, somewhat heuristically, to solve this problem.

Reply:
Well, the filtered samples are still filtered at the genotype level, but admittedly most users and tools don't really look at that.  I'd argue that the MT joint calling case is different from tumor multi-sample (especially single individual, multi-sample) because there's a not insignificant probability of recurrence in the MT case, whereas tumor variants from the same individual are almost certainly identical by descent and should be filtered similarly.  MT calls will have enough depth that there's not much power to be gained from aggregating strand bias observations.  If you have specific examples of Mutect2 logic that might be helpful, then I'm happy to take a look.